server.py

```diff
@@ -0,0 +1,352 @@
+import flask
+from flask import Flask, json
+import waitress
+
+from datetime import datetime, timedelta
+
+from core import Controller
+from models import FoodCategory, FoodDetails, CustomerDetails, CustOrderSelection, CustOrderStatus
+
+
+not_found = {"message": "Not Found"}
+success = {"message": "Success"}
+failed = {"message": "Failed"}
+
+ 
+def create_app(fos):
+    """
+    Creates the server app.
+    """
+    app = Flask('Food Ordering System')
+
+    ### employee routes
+    @app.route('/employee/add_food_category', methods=['POST'])
+    def add_food_category():
+        """
+        >> curl -H "Content-Type: application/json" -X POST -d "{\"name\":\"drinks\"}" "localhost:8080/employee/add_food_category"
+        """
+        name = flask.request.json["name"]
+        add = fos.add_food_category(name)
+        return json.dumps("{}".format(success if add else failed))
+    
+    @app.route('/employee/add_food_details', methods=['POST'])
+    def add_food_details():
+        """
+        >> curl -H "Content-Type: application/json" -X POST -d "{\"category_id\":4, \"food_name\":\"vodka\", \"price\":2}" "localhost:8080/employee/add_food_details"
+        """
+        category_id = flask.request.json["category_id"]
+        food_name = flask.request.json["food_name"]
+        price = flask.request.json["price"]
+        add = fos.add_food_details(category_id, food_name, price)
+        return json.dumps("{}".format(success if add else failed))
+
+    @app.route('/employee/add_delivery_person', methods=['POST'])
+    def add_delivery_person():
+        """
+        >> curl -H "Content-Type: application/json" -X POST -d "{\"delivery_person_name\":\"test\", \"delivery_person_phone\":11}" "localhost:8080/employee/add_delivery_person"
+        """
+        delivery_person_name = flask.request.json["delivery_person_name"]
+        delivery_person_phone = flask.request.json["delivery_person_phone"]
+        add = fos.add_delivery_person(delivery_person_name, delivery_person_phone)
+        return json.dumps("{}".format(success if add else failed))
```

Comment:
Return the added delivery person resource instead, in JSON.

Reply:
Returned JSON